.env.example

```
@@ -221,22 +221,16 @@ TINYAUTH_LDAP_AUTHKEY=
-TINYAUTH_EXPERIMENTAL_TAILSCALE_HOSTNAME=
-# Tailscale auth key.
-TINYAUTH_EXPERIMENTAL_TAILSCALE_AUTHKEY=
-# Use ephemeral Tailscale node.
-TINYAUTH_EXPERIMENTAL_TAILSCALE_EPHEMERAL=false
-# Enable Tailscale Funnel.
-TINYAUTH_EXPERIMENTAL_TAILSCALE_FUNNEL=false
-# Listen on the Tailscale address instead of standard address.
-TINYAUTH_EXPERIMENTAL_TAILSCALE_LISTEN=false
+TINYAUTH_TAILSCALE_ENABLED=false
+# Tailscale API token.
+TINYAUTH_TAILSCALE_APITOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TINYAUTH_TAILSCALE_APITOKEN key should go before the TINYAUTH_TAILSCALE_ENABLED key

(UnorderedKey)